codejam-virtual-keyboard/src/js/lib/is.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  object: (duck) => typeof duck === 'object'
+    && duck !== null,
+
+  primitive: (duck) => typeof duck === 'number'
```

Comment:
I think if something `!isObject` it `isPrimitive`. It is not needed these checks.